collect_app/src/main/res/layout/dialogbox_layout.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="fill_parent"
+    android:layout_height="fill_parent"
```

Comment:
Please use "match_parent" instead of "fill_parent" as the second option is deprecated starting from API Level 8.